src/Ucommerce.Avenue.Umbraco/Controllers/CategoryController.cs

```diff
@@ -104,7 +103,7 @@ private IList<ProductViewModel> MapProductsInCategories(Category category)
 
             var subCategories = CatalogLibrary.GetCategories(category.Categories);
             var products =
-                SearchLibrary.GetProductsFor(category.Categories.Append(category.Guid).ToList(), facetsForQuerying);
+                CatalogLibrary.GetProductsFor(category.Categories.Append(category.Guid).ToList(), facetsForQuerying.ToFacetDictionary());
```

Comment:
Why does it require `.ToList()`? Could we make the API more friendly by changing parameter type to `IEnumerable<Guid>`?

Reply:
It's actually harder than just changing the type.
We need to add a way to handle `IEnumrable` in our API.
So if we really want this, it's going to be another PR